gemrb/core/Scriptable/Door.cpp

```diff
@@ -116,12 +116,12 @@ void Door::ToggleTiles(int State, int playsound)
 	if (State) {
 		state = !closedIndex;
 		if (playsound && !OpenSound.IsEmpty()) {
-			core->GetAudioDrv()->PlayRelative(OpenSound, SFX_CHAN_ACTIONS);
+			core->GetAudioDrv()->Play(OpenSound, SFX_CHAN_ACTIONS, toOpen[0], GEM_SND_SPATIAL);
 		}
 	} else {
 		state = closedIndex;
 		if (playsound && !CloseSound.IsEmpty()) {
-			core->GetAudioDrv()->PlayRelative(CloseSound, SFX_CHAN_ACTIONS);
+			core->GetAudioDrv()->Play(CloseSound, SFX_CHAN_ACTIONS, toOpen[0], GEM_SND_SPATIAL);
```

Comment:
shouldn't it be 1 instead of 0 here and below?

Reply:
Not exactly sure, looked at the docs and it says that it's one of two door points (both sides?), and then I was thinking if there are examples of points with huge distances, or if it is neglectable and just pick any of the two.

Reply:
It's for when it's open and when it's closed, but you're right, it is usually close enough that it probably makes no difference.